src/main/java/com/jcabi/github/RtUser.java

```diff
@@ -139,6 +146,31 @@ public UserEmails emails() {
     }
 
     @Override
+    public List<Notification> notifications() throws IOException {
+        final List<Notification> notifs =
+            new LinkedList<Notification>();
+        final JsonResponse resp = this.github().entry().uri()
+            .path("notifications")
+            .back()
+            .fetch()
+            .as(JsonResponse.class);
+        final JsonArray jsnnotifs = resp.json().readArray();
+        for (final JsonValue jsnnotif : jsnnotifs) {
+            final JsonObject notifobj = (JsonObject) jsnnotif;
+            final Notification notif = new RtNotification(
+                Long.parseLong(notifobj.getString("id"))
+            );
+            notifs.add(notif);
+        }
+        return notifs;
+    }
+
+    @Override
+    public void markAsRead(final Date lastread) {
+        // Will be implemented later
```

Comment:
I don't think so - there is a todo in the header:

```
/**
 * Github user.
 *
 * @author Yegor Bugayenko (yegor@tpc2.com)
 * @version $Id$
 * @since 0.1
 * @todo #913:30min Implement operations RtUser.markAsRead().
 */
@Immutable
@Loggable(Loggable.DEBUG)
@EqualsAndHashCode(of = { "ghub", "request" })
final class RtUser implements User {
```